frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/url_cell/url_cell.dart

```diff
@@ -299,7 +299,7 @@ class _VisitURLAccessoryState extends State<_VisitURLAccessory>
   void onTap() {
     var content =
         widget.cellContext.getCellData(loadIfNotExist: false)?.content ?? "";
-    if (!content.contains('http://')) {
+    if (!content.startsWith('http://') && !content.startsWith('https://')) {
```

Comment:
It would be better for performance if you could do a single pass to check that the string is `http` and not `https`. You could do that with a regular expression or your function. Right now, this is O(2n). Not that it makes a huge difference.